platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -521,13 +521,15 @@ public void onSurfaceTextureUpdated(SurfaceTexture surface) {
   @CallSuper
   protected void onDetachedFromWindow() {
     super.onDetachedFromWindow();
-    mapZoomButtonController.setVisible(false);
+    if (mapZoomButtonController != null) {
+      mapZoomButtonController.setVisible(false);
+    }
   }
 
   // Called when view is hidden and shown
   @Override
   protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
-    if (isInEditMode()) {
+    if (isInEditMode() || mapZoomButtonController == null) {
```

Comment:
I'm not seeing the added benefit here for null checking as:

```
boolean mapboxZoomButtonControllerIsNotNull = mapbooxZoomButtonController == null;
if(isInEditMode() || mapboxZoomButtonControllerIsNotNull){
   ///
}
```

imo this is harder to read as a simple null check, happy to hear the thoughts of @mapbox/android on this.